### PR TITLE
Pr/quick fixes

### PR DESCRIPTION
**Pull request by felixportuphy**

### CODE REVIEW

File: cypress/integration/tests/Profile/amali-userprofile.spec.js [hunk 1]
```diff
@@ -1,87 +1,145 @@
...
+    ).as('statistics')
+    
+    cy.get('a[href*="/profile"]').click()
+    cy.get('.ae-dropdown').click({ force: true })
+    cy.get('.v-list-item__title').contains('This Week').click().then(()=>{
+      cy.wait('@statistics').then(({ response }) => {
+        cy.get('.ae-value').eq(0).should('contain.text', response.body.data.summary.max_speed)
+        cy.get('.ae-value').eq(1).should('contain.text', response.body.data.summary.total_distance)
+        cy.get('.ae-value').eq(2).should('contain.text', response.body.data.summary.total_ride_duration)
+      })
```
Comment (JerryHukpati) [inline cypress/integration/tests/Profile/amali-userprofile.spec.js:101]: dynamic selectors

File: cypress/integration/tests/Profile/amali-userprofile.spec.js [hunk 2]
```diff
@@ -1,87 +1,145 @@
...
       })
+    cy.get('.ae-avatar')
+      .find('img')
+      .invoke('attr', 'src')
+      .should('include', '/profile-images/')
+      cy.get('.rounded-circle').eq(2).should('be.visible')
+  })
 
-      cy.go('back')
+  it('Should filter statistics', () => {
```
Comment (JerryHukpati) [inline cypress/integration/tests/Profile/amali-userprofile.spec.js:88]: test name not descriptive

File: cypress/integration/tests/Profile/amali-userprofile.spec.js [hunk 3]
```diff
@@ -1,87 +1,145 @@
...
+        cy.get('.ae-value').eq(2).should('contain.text', response.body.data.summary.total_ride_duration)
+      })
+    })
+  
+    cy.get('.ae-dropdown').click()
+    cy.get('.v-list-item__title').contains('This Year').click({force:true}).then(()=>{
+      cy.wait('@statistics').then(({ response }) => {
+        cy.get('.ae-value').eq(0).should('contain.text', response.body.data.summary.max_speed)
+        cy.get('.ae-value').eq(1).should('contain.text', response.body.data.summary.total_distance)
+        cy.get('.ae-value').eq(2).should('contain.text', response.body.data.summary.total_ride_duration)
```
Comment (JerryHukpati) [inline cypress/integration/tests/Profile/amali-userprofile.spec.js:119]: dynamic selectors. do you have to do all these assertions for one test case?

File: cypress/integration/tests/Single Ride/amali-single-ride.spec.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  //     'background-color',
+  //     'rgb(66, 181, 70)',
+  //   )
+
+  //   cy.wait('@createSummaryVideo')
+  //   cy.get('#video-map-icon').parent().click()
+  //   cy.get('.ae-no-summary-video-view')
+  //     .find('p')
+  //     .should('have.text', ' The video summary creation is in process. ')
+  // })
```
Comment (JerryHukpati) [inline cypress/integration/tests/Single Ride/amali-single-ride.spec.js:139]: remove commented out code

**Review by JerryHukpati — CHANGES_REQUESTED** [review]

File: cypress/integration/tests/CheckEmail/amali-checkemail.spec.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    cy.url().should('include','/sign-up')
+    cy.get('.mx-auto').should('include.text','Create new account')
+    
+  })
+
+  it('check unverified email', () => {
+    cy.get('#login_email').type(users.unverified_email)
+    cy.get('#submit_button').click()
+
+    cy.wait('@checkUser', { timeout: 5000 }).then((success) => {
```
[the remark this answers is not in the document]
Comment (felixportuphy) [reply cypress/integration/tests/CheckEmail/amali-checkemail.spec.js:59]: Done

File: cypress/integration/tests/Profile/amali-userprofile.spec.js (hunk 2 above)
Comment (felixportuphy) [reply cypress/integration/tests/Profile/amali-userprofile.spec.js:88]: Done

File: cypress/integration/tests/Single Ride/amali-single-ride.spec.js (hunk 4 above)
Comment (felixportuphy) [reply cypress/integration/tests/Single Ride/amali-single-ride.spec.js:139]: Done

**Review by JerryHukpati — APPROVED** [review]

**Pull request merged**